index.html

```diff
@@ -157,63 +179,69 @@ <h2>Multi-Post Stories</h2>
-            </ul>
+        <div class="first-half">
+            <div class="title">
+                <h2>About <br> Myself </h2>
+            </div>
+            <div class="para1">
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+                    my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                    contact me.</p>
+            </div>
+            <div class="text">
+                <p>LET'S CONNECT</p>
+            </div>
+            <div class="icons">
+                <ul>
+                    <li><a href="#"><img src="./images/github.png" alt="github-page"></a></li>
+                    <li><a href="#"><img src="./images/Linkedin.png" alt="linkedin-page"></a></li>
+                    <li><a href="#"><img src="./images/twitter.png" alt="twitter-page"></a></li>
+                    <li><a href="#"><img src="./images/Vector.png" alt="vector-page"></a></li>
+                </ul>
+            </div>
+            <div class="buttons">
+                <button type="submit">Get My Resume</button>
+            </div>
         </div>
-        <div class="framework">
-            <ul>
-                <li>
-                    <h2>Skills</h2>
-                </li>
-                <li><a href="#"><img src="./images/arrowside.jpg" alt="arrow-side"></a></li>
-            </ul>
+        <div class="second-half">
+            <div class="lang">
+                <ul>
+                    <li>
+                        <h2>Languages</h2>
+                    </li>
+                    <li><img src="./images/arrowdown.jpg" alt="arrow-down"></li>
+                </ul>
+            </div>
+            <div class="tech">
+                <div class="javascript">
+                    <img src="./images/javascript.png" alt="javascript">
+                    <p>Javascript</p>
+                </div>
+                <div class="html">
+                    <img src="./images/html.png" alt="html">
+                    <p>HTML</p>
+                </div>
+                <div class="css">
+                    <img src="./images/css.png" alt="css">
+                    <p>CSS</p>
+                </div>
+            </div>
+            <div class="framework">
+                <ul>
+                    <li>
+                        <h2>Frameworks</h2>
+                    </li>
+                    <li><a href="#"><img src="./images/arrowside.jpg" alt="arrow-side"></a></li>
+                </ul>
+            </div>
+            <div class="framework">
+                <ul>
+                    <li>
+                        <h2>Skills</h2>
+                    </li>
+                    <li><a href="#"><img src="./images/arrowside.jpg" alt="arrow-side"></a></li>
+                </ul>
+            </div>
         </div>
 
     </div>
```

Comment:
[OPTIONAL]

Your project is approved. But it would be nice if you fix the size of your `inputs` and `text-areas`
to follow the Figma template as much as possible.

| Your Project | Figma Template |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/28109626/154724635-232dabb8-cb1f-4ae6-bfb1-46b10f441c9e.png) | ![image](https://user-images.githubusercontent.com/28109626/154724769-c732a53d-ee7d-4e97-a5af-19ddf95c059e.png) |